extension/payment/mockedRules/NL.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @param {SDKContext} context
+ * @param {PaymentMethodInput} input
+ * @returns {Promise<PaymentMethod[]>|PaymentMethod[]}
+ */
+module.exports = async (context, input) => {
+  if (!input.checkout.paymentAddress || input.checkout.paymentAddress.countryCode !== 'DE') {
```

Comment:
Why paymentAddress ? what is it? I will rename back to billingAddress

  What will happen if if (!input.checkout.paymentAddress will be truthy.

I guess paymentAddress.countryCode !== 'DE') { is wrong here. Should be NL and checked 1 more time. Otherwise rules will apply to any country code